### DISCUSSION TITLE
[question] How to read various dimensions after creating stairs?

### DISCUSSION THREAD

**Discussion by mariusa** [blackears/blenderStairs#8] [Announcements]
Hi,

This is a question (more related to Blender I guess), not an issue. Hope others will find it useful too.

After creating the stairs, how can one measure various dimensions? 
For example: https://diy.stackexchange.com/questions/171461/staircase-design-review

More specifically, on curved stairs I'd like to learn the walk line at 2/3, 3/4 and 4/4 of the stair width.

Thanks!

I did try the measure tool, but the dimensions vary depending on zoom level
https://docs.blender.org/manual/en/latest/editors/3dview/toolbar/measure.html
https://projects.blender.org/blender/blender/issues/67596

**Comment by blackears** [comment]
If you select the  stairs object and then press the N key to open the side panel, the Items tab will have a section listing the dimensions of the selected object.

![image](https://github.com/blackears/blenderStairs/assets/6724608/869dd2d7-0873-44a6-888d-8857acf4e102)

**Comment by mariusa** [comment]
Thanks! Is there a way to select individual steps, and see the dimensions of that? I couldn't see how to "ungroup" the stairs object.
